interface.go

```diff
@@ -64,6 +64,10 @@ type OutRequest struct {
 	// client sweep tx.
 	SweepConfTarget int32
 
+	// HtlcConfTarget specifies the number of confirmations we require for
+	// on chain loop out htlcs.
+	HtlcConfTarget int32
```

Comment:
related to the discussion on the CLI flag name: why do we call it conf target here? Make it sound more like a fee estimation param, maybe we should be consistent in using something like `HtlcRequiredConfs`?

Reply:
oh yeah def, I think my mind just saw sweep target and went with it.

Reply:
should this one be renamed as well?